plugin-template

```diff
@@ -48,6 +48,7 @@ DEFAULT_SETTINGS = {
     'stable_branch': None,
     'travis_addtl_services': [],
     'travis_notifications': None,
+    'update_redmine': False,
```

Comment:
Does it makes sense to set it to True by default? Just because it's what it does right now.

Reply:
With the errors you faced, I believe it is better to start as False, so the plugin developer is consciously enabling it

Reply:
@fao89, @dkliban , ok, however then the rollout of this change should flip it to True in the majority of the cases.